gr-fec/lib/reed-solomon/exercise.c

```diff
@@ -22,8 +22,6 @@
 #include "fixed.h"
 #define EXERCISE exercise_ccsds
 #elif defined(BIGSYM)
```

Comment:
Now that we've removed the `_int` functions, we don't need `BIGSYM` (this part of the `#elif` was now empty, and probably wouldn't build with `-DBIGSYM`).
```suggestion
```